### PR TITLE
Core: Use AYON settings

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
## Changelog Description
Core project settings conversion was removed.

## Additional info
Replace `"global"` with `"core"` and moved conversions into the code.

## Testing notes:
Things that should be validated.
- [x] Subset name is calculated correctly.
- [x] ExtractReview works.
- [x] ExtractBurnin works (different background and font colors).
- [x] ExtractThumbnail works (agains different colors etc.).

### CODE REVIEW

File: client/ayon_core/plugins/publish/extract_burnin.py [hunk 1]
```diff
@@ -116,11 +118,11 @@ def _get_burnins_per_representations(self, instance, src_burnin_defs):
...
 
             # Filter output definition by `burnin` represetation key
-            repre_linked_burnins = {
-                name: output
-                for name, output in burnin_defs.items()
-                if name in repre_burnin_links
-            }
+            repre_linked_burnins = [
+                burnin_def
+                for burnin_def in burnin_defs.items()
```
Comment (BigRoy) [inline client/ayon_core/plugins/publish/extract_burnin.py:123]: Are you sure this should still be `.items()`? Pretty sure this will currently fail.

Comment (iLLiCiTiT) [reply client/ayon_core/plugins/publish/extract_burnin.py:123]: There should not be `.items()` 🙂 
```suggestion
                for burnin_def in burnin_defs
```

**Review by tokejepsen — CHANGES_REQUESTED** [review]
```
Traceback (most recent call last):
  File "C:\Users\tokejepsen\AppData\Local\Ynput\AYON\dependency_packages\ayon_2402141620_windows.zip\dependencies\pyblish\plugin.py", line 527, in __explicit_process
    runner(*args)
  File "C:\Users\tokejepsen\ayon-core\client\ayon_core\plugins\publish\extract_burnin.py", line 89, in process
    self.main_process(instance)
  File "C:\Users\tokejepsen\ayon-core\client\ayon_core\plugins\publish\extract_burnin.py", line 197, in main_process
    burnins_per_repres = self._get_burnins_per_representations(
  File "C:\Users\tokejepsen\ayon-core\client\ayon_core\plugins\publish\extract_burnin.py", line 117, in _get_burnins_per_representations
    "burnin_defs.keys(): {}".format(burnin_defs.keys())
AttributeError: 'list' object has no attribute 'keys'
```

**Review by jakubjezek001 — APPROVED** [review]
LGTM

**Pull request merged**